File: apps/claimly-node-js-admin/src/api/claim/Claim.ts
import { ClaimStatus } from "../claimStatus/ClaimStatus";
import { Customer } from "../customer/Customer";

export type Claim = {
  amount: number | null;
  claimStatus?: ClaimStatus | null;
  createdAt: Date;
  customer?: Customer | null;
  date: Date | null;
  description: string | null;
  id: string;
  updatedAt: Date;
};

File: apps/claimly-server/src/APIs/Claim/Dtos/Claim.cs
namespace Claimly.APIs.Dtos;

public class Claim
{
    public double? Amount { get; set; }

    public string? ClaimStatus { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? Customer { get; set; }

    public DateTime? Date { get; set; }

    public string? Description { get; set; }

    public string Id { get; set; }

    public string? Policy { get; set; }

    public DateTime UpdatedAt { get; set; }
}
